content/doc/book/system-administration/FIPS-140.adoc

```diff
@@ -0,0 +1,54 @@
+---
+layout: section
+---
+ifdef::backend-html5[]
+ifndef::env-github[:imagesdir: ../../resources/managing]
+:notitle:
+:description:
+:author:
+:email: jenkinsci-users@googlegroups.com
+:sectanchors:
+:toc: left
+endif::[]
+
+= FIPS-140
+
+[IMPORTANT]
+----
+support blurb
+----
+
+
+== Jenkins 
+
+It may be possible to run Jenkins in a FIPS-140 compliant nature when the <<managing/system-properties#jenkins-security-FIPS140-COMPLIANCE, compliance flag>> is enabled, the servlet container is appropriately configured, the JVM is appropriately configured, and the host OS is appropriately configured.
+How to configure the servlet container, JVM and host are out of scope of the Jenkins community project as this is a complex area with many pitfalls and gotchas.
+Some Jenkins features may not work or be disabled.
+
+== Plugins
+
+Plugins may or may not honour a request to run in FIPS-140 compliance mode.
+Before you install or upgrade any plugin, you should make checks of the plugin's code to ensure it adheres to the FIPS-140 standard.
+
+== What FIPS-140 mode does
+
+Enabling <<managing/system-properties#jenkins-security-FIPS140-COMPLIANCE, FIPS mode>> provides a hint to Jenkins and any plugins that have opted in that they should prefer cryptographic algorithms that *may*.footnote:[Algorithms are not approved, rather an implementation of an algorithm is.  
+However, the implementation used at runtime depends on the JVM, JVM configuration, and the host OS.
+As this is outside the scope of the Jenkins project the algorithms targeted are known to be available from at least one link:https://csrc.nist.gov/projects/cryptographic-module-validation-program/validated-modules/search[FIPS compliant provider], namely the link:https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/3514[BouncyCastle FIPS library].] be FIPS approved.
+This may mean that some features are disabled entirely, or even may use a less secure (but compliant) form of cryptography than normal.
```

Comment:
```suggestion
This may mean that some features are disabled entirely, or may use a less secure (but compliant) form of cryptography than normal.
```